package.json

```diff
@@ -22,7 +22,7 @@
         "@commitlint/cli": "^18.4.3",
         "@commitlint/config-conventional": "^18.4.3",
         "@testing-library/jest-dom": "^6.1.5",
-        "@testing-library/react": "^14.1.2",
+        "@testing-library/react": "^14.3.1",
```

Comment:
@BRIHMAT-Naoui ,It would be better for us to avoid updating packages If the pull request is not intended for that.